Kernel/Syscalls/futex.cpp

```diff
@@ -105,9 +105,7 @@ ErrorOr<FlatPtr> Process::sys$futex(Userspace<Syscall::SC_futex_params const*> u
 
     switch (cmd) {
     case FUTEX_WAIT:
-    case FUTEX_WAIT_BITSET:
-    case FUTEX_REQUEUE:
-    case FUTEX_CMP_REQUEUE: {
```

Comment:
am I missing something here? in LibC, you still pass the futex op through unchaged, which means we can still get FUXTEX_REQUEUE and FUTEX_CMP_REQUEUE in this swtich statement. 

These two options are now not covered at all in this swtich, is that intentional? If it is, please add an empty case for them with a `break` statement.

Reply:
I'm probably making (yet another) mistake here heh. I'll take another look

Reply:
Ah yes it is intentional - This part just deals with the specifics of pulling out the timeout from calls that supply it - the issue being some calls which don't supply it were listed. 

I can add some empty cases, however I think there actually might already be some other types of call not listed here. This is basically being used as a possibly nicer syntax version of an if statement